tokenspice.ini

```diff
@@ -2,7 +2,7 @@
 [general]
 
 NETWORK = ganache
-ARTIFACTS_PATH = /home/trentmc/code/contracts/artifacts
+ARTIFACTS_PATH = contracts/artifacts
```

Comment:
Contracts are typically _not_ going found in a subdirectory of `tokenspice` directory. They'll be found inside the contracts repo that was git-cloned. (As discussed in the call, we _will_ move it to be a subdirectory of tokenspice at some point. But that requires a bunch more work and machinery to be in place, it's outside the scope of this issue.)

Perhaps make the default `~/contracts/artifacts/` or `../contracts/artifacts`

Reply:
Hello, Trent! I've tried to adjust the workflow such that the contracts repo is outside the tokenspice repo both as 
`~/contracts/artifacts/` and `../contracts/artifacts` as you have suggested. 

After several hours of debugging I discovered why the workflow was acting unpredictably. It turns out there's an inherent limitation with the [Github Action](https://github.com/actions/checkout) that is used to clone repos —  a user must clone other repos within $GITHUB_WORKSPACE ( in this case, it's the tokenspice directory. ) 

There have been several about this over the past months, see here:
https://github.com/actions/checkout/pull/388
https://github.com/actions/checkout/issues/197

I'm currently exploring possible hacks to get around this like using symlinks that point to something outside the repo root.